### PR TITLE
Rjdev

### PR DESCRIPTION
**Pull request by rjovelin**
this PR is to make barcodex compliant with barcodex-rs

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,26 +24,23 @@ Parameters
...
 | --r1_in | Path(s) to FASTQ(s) containing read 1   | required                |
-| --r1_out | Path to output FASTQ 1   | required                |
 | --r2_in | Path(s) to FASTQ(s) containing read 2   | optional              |
-| --r2_out | Path to output FASTQ 2   | optional              |
 | --r3_in | Path(s) to FASTQ(s) containing UMIs for paired end with non-inline UMIs | optional              |
 | --pattern1 | pattern or regex for extracting UMIs in read 1   | optional              |
 | --pattern2 | pattern or regex for extracting UMIs in read 2   | optional              |
-| --prefix | the prefix used for statistics output | optional              |
+| --prefix | Specifies the start of the output files | required              |
 | --data | paired or single end sequencing   | required              |
```
Comment (apmasell) [inline README.md:32]: I've dropped `--data` from the Rust version. I'm not sure if you want to do the same.

File: README.md [hunk 2]
```diff
@@ -24,26 +24,23 @@ Parameters
...
 | --separator | String separating the UMI sequence in the read name   | required              |
 | --umilist | Path to file with valid UMIs | optional              |
 | --inline | UMIs inline with reads or not | optional              |
```
Comment (apmasell) [inline README.md:35]: This has been switched to sub commands `inline` and `separate`. Again, I'm not sure if you want to keep it that way.

**Review by apmasell — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (rjovelin) [reply README.md:32]: I will drop --data to make the python version fully compliant

File: README.md (hunk 2 above)
Comment (rjovelin) [reply README.md:35]: I will add the sub commands to make the python version fully compliant

**Pull request merged**